solution/project/azure.py

```diff
@@ -5,17 +5,21 @@
 
 ALLOWED_HOSTS = [os.environ['WEBSITE_HOSTNAME']] if 'WEBSITE_HOSTNAME' in os.environ else []
 
+CSRF_TRUSTED_ORIGINS = ['https://'+ os.environ['WEBSITE_HOSTNAME']] if 'WEBSITE_HOSTNAME' in os.environ else []
+
 hostname = os.environ['DBHOST']
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
         'NAME': os.environ['DBNAME'],
         'HOST': hostname + ".postgres.database.azure.com",
-        'USER': os.environ['DBUSER'] + "@" + hostname,
+        'USER': os.environ['DBUSER'],
         'PASSWORD': os.environ['DBPASS'] 

```

Comment:
For Postgres Flexible (which is what we recommend in Learning Path) doesn't require at hostname part. Added commend below in the chance someone tries to use Postgres Single (which we discourage against).